digitalocean-1-click/withoutpg.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+
+set -ex
```

Comment:
I don't think we need two distinct images here, because it's trivial to turn a "withpg" image into a "withougpg" by setting "EDGEDB_SERVER_BACKEND_DSN" in `/etc/edgedb/env`

Reply:
changed